sway-lib-std/src/fixed_point.sw

```diff
@@ -0,0 +1,130 @@
+library fixed_point;
+//! A wrapper around U128 type for a library for Sway for mathematical functions operating with signed 64.64-bit fixed point numbers. 
+
+use ::u128::U128;
+use ::assert::assert;
+
+pub struct UFP64 {
```

Comment:
Idk about this name. It doesn't really roll off the tongue that well although I have no suggestions for how to make it better

Reply:
Ah actually these are fixed point, not floating point, so `FP` might make sense.